firebase-installations-interop/src/main/java/com/google/firebase/installations/FirebaseInstallationsApi.java

```diff
@@ -28,26 +28,23 @@
  */
 public interface FirebaseInstallationsApi {
 
-  /** Specifies the refresh options to get a FIS AuthToken. */
-  @IntDef({FORCE_REFRESH, DO_NOT_FORCE_REFRESH})
-  @Retention(SOURCE)
-  @interface RefreshAuthTokenOption {}
   /**
-   * AuthToken is forcefully refreshed on calling the {@link
-   * FirebaseInstallationsApi#getAuthToken()}.
+   * Specifies the options to get a FIS AuthToken. Default value of AuthTokenOption =
+   * DO_NOT_FORCE_REFRESH.
```

Comment:
the default part is describing the method, not this option, so the javadoc should be on the method not this option

Reply:
Is this right now?

Reply:
Sorry, I wasn't clear. You had it correct (Javadoc above the `@interface` declaration), I just meant the "Default value of AuthTokenOption = DO_NOT_FORCE_REFRESH" should be above the method that uses it

Reply:
Thanks for the clarification. I corrected it.